georchestra-integration/externalized-accounts/src/main/java/org/geonetwork/security/external/integration/AbstractGroupSynchronizer.java

```diff
@@ -211,7 +211,8 @@ private void deleteGoneGroups(final Map<String, GroupLink> currentLinks, final S
     }
 
     private void updateLabelTranslations(CanonicalGroup canonical, Group group) {
-        final String newName = canonical.getName();
+        final String newName = Optional.ofNullable(canonical.getOrgTitle()).map(String::isBlank).orElse(true)
+            ? canonical.getName() : canonical.getOrgTitle();
```

Comment:
I don't find the lambda mix with ternary operator very readable, and was wondering if a simpler construct would be more appropriate here, but I can live with it :)